### PR TITLE
Samuel Giles Potluck 

### PR DESCRIPTION
**Pull request by SKG360**
Iterations 1 and 2 (in part)

struggled with wrong number of arguments errors in test methods

### CODE REVIEW

**Comment by SKG360** [comment]
submitted

File: test/dish_test.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+require "./lib/dish"
+require "./lib/potluck"
+
+class DishTest < Minitest::Test
+
+  def test_if_it_exists
+    dish = Dish.new("Conscious Salad", :appetizer)
+    assert_instance_of Dish, dish
+  end
+
```
Comment (BrianZanti) [inline test/dish_test.rb]: You are missing tests for the methods name and category

File: test/potluck_test.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+  def test_if_it_exists
+    potluck = Potluck.new
+    assert_instance_of Potluck, potluck
+  end
+
+  def test_if_it_adds_dishes
+    potluck = Potluck.new
+    couscous_salad = Dish.new("Couscous Salad", :appetizer)
+    assert_equal "Couscous Salad", potluck.add_dish("Couscous Salad")
```
Comment (BrianZanti) [inline test/potluck_test.rb]: We don't want to pass the string "Couscous Salad" to the add_dish method, we want to pass the Dish object `couscous_salad`

File: lib/potluck.rb [hunk 3]
```diff
@@ -0,0 +1,27 @@
+require "./lib/dish"
+
+class Potluck
+
+  attr_accessor :date, :appetizer
+
+  def initialize
+    @date = date
```
Comment (BrianZanti) [inline lib/potluck.rb]: The date needs to be passed in to initialize

File: lib/potluck.rb [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    @appetizer = appetizer
+    @dishes = []
+  end
+
+  def add_dish
```
Comment (BrianZanti) [inline lib/potluck.rb]: This should take an argument

**Pull request closed without merging**